File: src/codes/Users_Manual/modulation/square_0.cpp
//~#include<autd3.hpp>
//~int main() {
//~using namespace autd3;
Square(150 * Hz, SquareOption{
                     .low = 0x00,
                     .high = 0xFF,
                     .duty = 0.5f,
                     .sampling_config = SamplingConfig::FREQ_4K,
                 });
//~return 0; }

File: src/codes/Users_Manual/modulation/square_0.rs
# use autd3::prelude::*;
# fn main() {
# let _ = 
Square {
    freq: 150 * Hz,
    option: SquareOption {
        low: u8::MIN,
        high: u8::MAX,
        duty: 0.5,
        sampling_config: SamplingConfig::FREQ_4K,
    },
};
# }